### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by WilsonCheng-sy**

### CODE REVIEW

**Review by suvincent — APPROVED** [review]

File: doc/watir-installation.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# chrome install for wsl
```
Comment (chinyakao) [inline doc/watir-installation.md:1]: Nice doc folder to keep the fighting path.

**Review by chinyakao — COMMENTED** [review]
@WilsonCheng-sy @suvincent @RayhungKao

File: app/infrastructure/database/migrations/001_papers_create.rb [hunk 2]
```diff
@@ -14,9 +14,9 @@
       String      :date, unique: false, null: true
       String      :organization, unique: false, null: true
       String      :author, unique: false, null: true
-      String     :publication_id, unique: true, null: false
+      String :publication_id, unique: true, null: false
 
-      Integer     :citedby
+      Integer :citedby
```
[the remark this answers is not in the document]
Comment (WilsonCheng-sy) [reply app/infrastructure/database/migrations/001_papers_create.rb:20]: For this, we also feel it is weird, we first run _rake quality:all_ and nothing change. 
So we moving on to run rubocop for this file manually but seems rubocop not suggest us to modify. (As the follow picture)

However, we do think it is some typo we made, so we would manually align the code.

image

File: app/infrastructure/database/migrations/002_publications_create.rb [hunk 3]
```diff
@@ -5,10 +5,10 @@
 Sequel.migration do
   change do
     create_table(:publications) do
-      primary_key  :id
+      primary_key :id
 
       # pid equals to publication_id
-      String      :pid, unique: true, null: false
```
[the remark this answers is not in the document]
Comment (WilsonCheng-sy) [reply app/infrastructure/database/migrations/002_publications_create.rb:11]: Just as the above

File: doc/watir-installation.md (hunk 1 above)
Comment (WilsonCheng-sy) [reply doc/watir-installation.md:1]: Thanks haha

**Pull request merged**